conda_smithy/github.py

```diff
@@ -220,21 +220,21 @@ def configure_github_team(meta, gh_repo, org, feedstock_name):
     # Try to get team or create it if it doesn't exist.
     team_name = feedstock_name
     current_maintainer_teams = list(gh_repo.get_teams())
-    team = next(
+    fs_team = next(
```

Comment:
why?

Reply:
The `team` variable is used inside a loop below. I made this change to try and make the code clearer and more explicit.